iis-core/src/test/java/eu/dnetlib/iis/core/AbstractOozieWorkflowTestCase.java

```diff
@@ -97,14 +140,52 @@ protected void testWorkflow(String workflowPath, OozieWorkflowTestConfiguration
 		
 		String jobId = getJobId();
 		
+		
 		Status jobStatus = waitForJobFinish(jobId, configuration.getTimeoutInSeconds());
 		
-		assertJobStatus(jobId, jobStatus);
+		assertJobStatus(jobId, jobStatus, configuration.getExpectedFinishStatus());
+		
+		
+		Properties jobProperties = fetchJobProperties(jobId);
+		String workflowWorkingDir = jobProperties.getProperty("workingDir");
+		
+		WorkflowTestResult result = new WorkflowTestResult();
+		
+		importFilesFromHdfs(result, configuration.getOutputFilesToInclude(), workflowWorkingDir);
+		importAvroDatastoresFromHdfs(result, configuration.getOutputAvroDataStoreToInclude(), workflowWorkingDir);
+		
+		return result;
 	}
 	
 	
 	//------------------------ PRIVATE --------------------------
 	
+	private Properties fetchJobProperties(String jobId) {
```

Comment:
I'm thinking about moving it and other oozie job related methods to special service like OozieJobInfoService or something like that.